src/components/DisplaySetting.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import "../App.css";
+function DisplaySetting(props) {
+  // const [displaySettingComponent, setDisplaySettingComponent] = useState("none");
+  const { attributesName, handleOutputImagesAttributes } = props;
+  // const [attributesToHide, setAttributesToHide] = useState([]);
+
+  const [attributesToShow, setAttributesToShow] = useState(attributesName);
+  const handleDisplaySettingChange = (event) => {
+    const { name, value } = event.target;
+
+    attributesToShow.includes(name) === true
+      ? setAttributesToShow(
+          attributesToShow.filter((attribute) => {
+            if (attribute === name) return false;
+            else return true;
+          })
+        )
```

Comment:
write a function to include this logic instead of adding logic in ternary condition